internal/proxy/task_scheduler.go

```diff
@@ -177,18 +177,14 @@ func (queue *baseTaskQueue) Enqueue(t task) error {
 	var id UniqueID
 	if t.CanSkipAllocTimestamp() {
 		ts = tsoutil.ComposeTS(time.Now().UnixMilli(), 0)
-		id, err = globalMetaCache.AllocID(t.TraceCtx())
```

Comment:
should we also remove  globalMetaCache.AllocID?

Reply:
yes, allocID is only used here, so I'll clean it up.